scrapers/urbandictionary.rb

```diff
@@ -21,6 +25,13 @@ def get_definitions(term)
     definitions.map { |m| parse_definition(term, m) }
   end
 
+  def get_random_definition()
+    uri = URL(DOMAIN+RANDOM)
```

Comment:
From what I've seen there's no such thing as an `URL` method or class, but there is an `URI` hence the correction